src/Nuke/RocketBoosterBuild.cs

```diff
@@ -105,10 +107,24 @@ public abstract class RocketBoosterBuild : NukeBuild
         [Parameter("The directory where coverage artifacts are to be dropped", Name = "Coverage")]
         public readonly AbsolutePath CoverageDirectory = Variable("Coverage") != null ? (AbsolutePath)Variable("Coverage") : RootDirectory / "coverage";
 
+        /// <summary>
+        /// prints the build information.
+        /// </summary>
+        public Target BuildVersion => _ => _
+            .Executes(() =>
+            {
+                Logger.Info("Building version {0} of {1} ({2}) using version {3} of Nuke.",
+                    GitVersion.NuGetVersionV2 ?? GitVersion.NuGetVersion,
```

Comment:
what about FullSemVer?

Reply:
This is just something I had in another personal project.  I am fine if you want to change it to `FullSemVer` I just felt like something was better than nothing.